task_manager_app/views.py

```diff
@@ -0,0 +1,111 @@
+from django.utils import timezone
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from task_manager_app.models import Task, SubTask
+from task_manager_app.serializer import TaskSerializer, CreateTaskSerializer, TaskDetailSerializer, SubTaskSerializer, \
+    CreateSubTaskSerializer
+
+
+
+class SubTaskListCreateView(APIView):
+    def get(self, request):
+        subtasks = SubTask.objects.all()
+        serializer = SubTaskSerializer(subtasks, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    def post(self, request):
+        task = CreateSubTaskSerializer(data=request.data)
+        if task:
+            if task.is_valid():
+                task.save()
+            return Response(status=status.HTTP_204_NO_CONTENT)
```

Comment:
Nne лишняя проверка. Экземпляр сериализатора, если нет ошибки, будет всегда существовать. К тому же неверный статус
```suggestion
        if serializer.is_valid():
            serializer.save()
            return Response(serializer.data, status=status.HTTP_201_CREATED)
```